smack-websocket/src/main/java/org/jivesoftware/smack/websocket/rce/RemoteWebsocketConnectionEndpoints.java

```diff
@@ -0,0 +1,60 @@
+/**
+ *
+ * Copyright 2020 Aditya Borikar
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smack.websocket.rce;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jivesoftware.smack.altconnections.HttpLookupMethod;
+import org.jivesoftware.smack.altconnections.HttpLookupMethod.LinkRelation;
+import org.jivesoftware.smack.c2s.internal.ModularXmppClientToServerConnectionInternal;
+import org.jivesoftware.smack.util.rce.RemoteConnectionEndpoint;
+import org.jivesoftware.smack.util.rce.RemoteConnectionEndpointLookupFailure;
+import org.jivesoftware.smack.xml.XmlPullParserException;
+
+public final class RemoteWebsocketConnectionEndpoints {
```

Comment:
To be in line with the other subclasses of `RemoteConnectionEndpoint`, this should be named `WebsocketRemoteConnectionEndpoint`.